src/listproperties/listpropertiesediting.ts

```diff
@@ -255,6 +259,21 @@ export interface AttributeStrategy {
  * Creates an array of strategies for dealing with enabled listItem attributes.
  */
 function createAttributeStrategies( enabledProperties: ListPropertiesConfig ) {
+	// FIXME: Check if we still need that
```

Comment:
This is needed, it maps key to actual html attribute value

Reply:
reverted